Makefile

```diff
@@ -252,25 +252,29 @@ ml4clean:
 	rm -f $(GENML4FILES) $(GENMLGFILES)
 
 depclean:
-	find . $(FIND_SKIP_DIRS) '(' -name '*.d' ')' -print | xargs rm -f
+	find . $(FIND_SKIP_DIRS) '(' -name '*.d' ')' -exec rm -f {} +
 
 cacheclean:
-	find theories plugins test-suite -name '.*.aux' -delete
+	find theories plugins test-suite -name '.*.aux' -exec rm -f {} +
 
 cleanconfig:
 	rm -f config/Makefile config/coq_config.ml myocamlbuild_config.ml dev/ocamldebug-coq dev/camlp5.dbg config/Info-*.plist
 
 distclean: clean cleanconfig cacheclean timingclean
 
 voclean:
-	find theories plugins test-suite \( -name '*.vo' -o -name '*.glob' -o -name "*.cmxs" -o -name "*.native" -o -name "*.cmx" -o -name "*.cmi" -o -name "*.o" \) -delete
-	find theories plugins test-suite -name .coq-native -empty -delete
+	find theories plugins test-suite \( -name '*.vo' -o -name '*.glob' -o -name "*.cmxs" \
+	-o -name "*.native" -o -name "*.cmx" -o -name "*.cmi" -o -name "*.o" \) -exec rm -f {} +
+	find theories plugins test-suite -name .coq-native -empty -exec rm -f {} +
```

Comment:
@jfehrle @gares why change this line? There was no call to `xargs` and the change is incorrect. The one on the line above also looks useless. Same question for `timingclean`.

Reply:
Ok, I see why @SkySkimmer suggested to use `-exec rm` instead of `-delete`. I don't know why, but here it is not correct as done currently because `.coq-native` is a directory.

Reply:
oops, let's open an issue

Reply:
Submitted PR https://github.com/coq/coq/pull/8497 to use `rm -rf`.